### PR TITLE
Ft user provide review 187570881

### PR DESCRIPTION
**Pull request by rodriguecyber**
### What does this PR do?

1.Adding new controler for user review product added

### Description of Task to be completed?

- A buyer should be able to provide a review on a product they have bought successfully, with feedback

### How should this be manually tested?
 1. Login as buyer: `http://localhost:3000/api/login/user` using `POST `
 body:{
"email": "",
  "password": ""
}
 2.Authorisation token must be in headers as Bearer
 3. create new review for product : `localhost:3000/api/review/create`  using `POST` method
 body:{
    "productId":2,
    "RatingValue":7,
    "review":""
}

### What are the relevant pivotal tracker stories
link to pivot tracker story: https://www.pivotaltracker.com/n/projects/2703469

### CODE REVIEW

File: src/controllers/reviewController/review.controller.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { Request, Response } from "express";
+import {reviewService} from '../../services/review.service.js'
+import { ReviewAttributes } from "models/review.js";
+
+class reviewController{
+
+    async addReview(req:Request,res:Response){
+      // const {userId,productId,review,RatingValue} = req.body
```
Comment (Balinda21) [inline src/controllers/reviewController/review.controller.ts]: Remove this comment

File: src/routes/roleRoutes.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // import { isRoleAdmin } from '../middleware/checkAdminRoleMiddleware.js';
```
Comment (Balinda21) [inline src/routes/roleRoutes.ts:1]: Remove this commented codes

**Review by Balinda21 — APPROVED** [review]
Well done @rodriguecyber, consider those changes

File: src/routes/reviewroute.ts [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+ *             schema:
+ *               type: object
+ *               properties:
+ *                 message:
+ *                   type: string
+ *        
+ */
+
+ reviewRouter.post('/review/create',isLoggedIn,BuyerMiddleware, reviewControllers.addReview)
+
```
Comment (kirengamartial) [inline src/routes/reviewroute.ts:67]: add new line

**Review by kirengamartial — APPROVED** [review]
the overall of code it looks good but consider the feedbacks

File: src/routes/roleRoutes.ts (hunk 2 above)
Comment (rodriguecyber) [reply src/routes/roleRoutes.ts:1]: is not for my changes

File: src/controllers/reviewController/review.controller.ts (hunk 1 above)
Comment (rodriguecyber) [reply src/controllers/reviewController/review.controller.ts]: done

**Review by DeliceLydia — APPROVED** [review]

**Pull request merged**